tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -404,41 +404,23 @@ func parseAction(cmd *cobra.Command, args []string, update bool) (*whisk.Action,
             }
         }
 
-        if flags.action.kind == "swift:3" || flags.action.kind == "swift:3.0" || flags.action.kind == "swift:3.0.0" {
-            action.Exec.Kind = "swift:3"
-        } else if flags.action.kind == "nodejs:6" || flags.action.kind == "nodejs:6.0" ||
-                flags.action.kind == "nodejs:6.0.0" {
-            action.Exec.Kind = "nodejs:6"
-        } else if flags.action.kind == "nodejs:default" {
-            action.Exec.Kind = "nodejs:default"
-        } else if flags.action.kind == "swift:default" {
-            action.Exec.Kind = "swift:default"
-        } else if flags.action.kind == "nodejs" {
-            action.Exec.Kind = "nodejs"
-        } else if flags.action.kind == "python" {
-            action.Exec.Kind = "python"
+        if len(flags.action.kind) > 0 {
```

Comment:
If a user enters a `--kind` value that is not supported, will the backend return an error when the request is sent for processing?

It would be fantastic if you can create a function that handles all the `--kind` processing and reference that in `parseAction()`.